presidio-image-redactor/tests/integration/test_qr_image_analyzer_engine_integration.py

```diff
@@ -0,0 +1,39 @@
+from presidio_image_redactor.entities import ImageRecognizerResult
+from tests.integration.methods import get_resource_image
+
+
+def test_given_qr_image_then_text_entities_are_recognized_correctly(
```

Comment:
very nice scenarios

Reply:
Should I add more complex/realistic scenarios?